src/components/SchedulePage/Schedule.js

```diff
@@ -3,17 +3,17 @@ import React from 'react';
 import { makeStyles } from '@material-ui/core/styles';
 import Typography from '@material-ui/core/Typography';
 import Container from '@material-ui/core/Container';
-import List from '@material-ui/core/List';
-import Divider from '@material-ui/core/Divider';
+// import List from '@material-ui/core/List';
+// import Divider from '@material-ui/core/Divider';
 
-import StickyTimeSlot from '../SchedulePage/StickyTimeSlot';
-import workshopSchedule from '../../data/WorkshopSchedule';
+// import StickyTimeSlot from '../SchedulePage/StickyTimeSlot';
+// import workshopSchedule from '../../data/WorkshopSchedule';
 
```

Comment:
As an alternative to commenting out code (which is hard to maintain), you could do the following. Add a line called

```suggestion

const scheduleAvailable = false;
```

Then, later change the `paddingBottom` line to

```
		paddingBottom: theme.spacing(scheduleAvailable ? 4 : 11)
```

and then change `<List ...> ... </List>` to

```
{scheduleAvailable && <List ...>
  ...
</List>}
```